tox.ini

```diff
@@ -35,8 +35,8 @@ commands =
     coverage report
 
 [testenv:lint]
-basepython = python3.7
+basepython = python3.8
 deps = flake8
 commands =
-    flake8 src tests setup.py
-    isort --recursive --check-only --diff src tests
+    ruff format --check src/ tests/
+    ruff check src/ tests/
```

Comment:
question: why not `.` instead of `src/ tests/`?

Reply:
No good reason! Changed in ea8b5cc